deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -344,6 +344,9 @@ def get(self, request, name, stage=None):
             basic_cluster_info = clusters_helper.get_cluster(request, env.get('clusterName'))
             capacity_info['cluster'] = basic_cluster_info
 
+        console_project_exist = environs_helper.get_nimbus_project(name)
+        console_project_env_url = environs_helper.get_console_project_env_url(name)
```

Comment:
same here the variable "name" here should be the nimbus project name associated with the stage.